README.md

```diff
@@ -46,14 +46,34 @@ Example for running this command given:
 - **build number**: 70
 - **build url**: https://travis-ci.org/030/bcbsn/builds/523263434
 
-```
-./bcbsn -keyString 1234:5678 -buildState INPROGRESS -gitCommit f76d123498a053c1789057a41d6c3fcvg8b49cd7 -owner 030 -repositoryName bcbsn -buildNumber 70 -buildURL https://travis-ci.org/030/bcbsn/builds/523263434
-```
-
 [![dockeri.co](https://dockeri.co/image/utrecht/bcbsn)](https://hub.docker.com/r/utrecht/bcbsn)
 
 To print the help message
 
 ```
 docker run utrecht/bcbsn:2.0.0 -h
 ```
+
+returns:
+
+```
+Usage of /usr/local/bcbsn:
```

Comment:
We don't now this path (`/usr/local/bcbsn`), you can install it somewhere else

Reply:
https://serverfault.com/a/629051/215599

Reply:
It's fine to leave it in, but like the answers you refer to it just a standard, you don't have to follow those. For all I care you install this package in /tmp 😉